pkg/steps/project_image.go

```diff
@@ -76,18 +72,44 @@ func (s *projectDirectoryImageBuildStep) run(ctx context.Context) error {
 
 	images := buildInputsFromStep(s.config.Inputs)
 	// If image being built is an operator bundle, use the bundle source instead of original source
-	if s.config.OperatorManifests != "" {
+	if strings.HasPrefix(string(s.config.To), api.BundlePrefix) {
```

Comment:
Can we write a function for this logic - to make it clearly labeled and obvious. Raw string manipulations scattered around made a lot of our `ci-operator` code rot over time in e.g. env vars

Reply:
And presumably a function to create the name from an index and prefix, and a test that `detect(wrap()) == true` and `unwrap(wrap(input)) == input` if there is an `unwrap`